contracts/interfaces/IIncurDebtTokemak.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity >=0.8.4;
+
+/// @notice simplified IncurDebt to work as a Tokemak controller
+/// @dev this contract must be given OHMDEBTOR on the treasury
+interface IIncurDebtTokemak {
+    /* ========== TOKEMAK FUNCTIONS ========== */
+
+    /**
+     * @notice deposits gOHM to use as collateral
+     * - msg.sender must be whitelisted
+     * - this contract must have been approved _amount of gOHM
+     * @dev will unwrap and hold as sOHM in this contract
+     * @param _gOHMAmount amount of gOHM
+     */
+    function deposit(uint256 _gOHMAmount) external;
+
+    /**
+     * @notice deposits gOHM to use as collateral
+     * - msg.sender must be whitelisted
+     * - _ohmAmount must be <= availableDebt
+     * @param _ohmAmount amount of OHM to borrow
+     * @param _to address that will receive borrowed OHM
+     */
+    function borrow(uint256 _ohmAmount, address _to) external;
+
+    /**
+     * @notice deposits gOHM to use as collateral
+     * - msg.sender must be whitelisted
+     * - msg.sender's OHM allowance must be >= _ohmAmount 
+     * @param _ohmAmount amount of OHM to borrow
+     */
+    function repay(uint256 _ohmAmount) external;
+
+    /**
+     * @notice gets available OHM to borrow for account
+     * @param _account account to inspect
+     * @return amount OHM available to borrow
+     */
+    function getAvailableToBorrow(address _account) external returns(uint256);
+
+    /* ========== MANAGEMENT FUNCTIONS ========== */
```

Comment:
We also need a way for Policy to set LTV. Currently we've discussed this being 100%, but I think it would be good to have flexibility in the future. This would meant hat borrows need to check LTV against how much the partner wants to borrow.